<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>731</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>731</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>731</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Burca's</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(200, 200, 200);</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>19</y>
         <width>321</width>
         <height>571</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>700</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 167, 125);
border-radius: 15px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-20</y>
          <width>300</width>
          <height>181</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="pixmap">
         <pixmap>logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>250</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Mongolian Baiti</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgba(100, 100, 100, 100);
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	pading:15px;
	background-color:rgb(30, 30, 30);
}
QLineEdit:hover {
	border: 2px solid rgb(55, 55, 55)
}
QLineEdit:focus {
	border: 2px solid rgb(255, 255, 255);
	color: rgb(200, 200, 200);
}</string>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>USER</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>310</y>
          <width>280</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Mongolian Baiti</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	color: rgba(100, 100, 100, 100);
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	pading:15px;
	background-color:rgb(30, 30, 30);
}
QLineEdit:hover {
	border: 2px solid rgb(55, 55, 55)
}
QLineEdit:focus {
	border: 2px solid rgb(255, 255, 255);
	color: rgb(200, 200, 200);
}</string>
        </property>
        <property name="maxLength">
         <number>16</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>SENHA</string>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>281</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 10px;
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
	color:rgb(35, 35, 35);
}</string>
        </property>
        <property name="text">
         <string>ENTRAR</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(30, 30, 30);</string>
        </property>
        <property name="text">
         <string>CADASTRO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>490</y>
          <width>281</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 10px;
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed{
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
	color:rgb(35, 35, 35);
}</string>
        </property>
        <property name="text">
         <string>VOLTAR</string>
        </property>
       </widget>
       <zorder>lineEdit</zorder>
       <zorder>lineEdit_2</zorder>
       <zorder>pushButton</zorder>
       <zorder>label</zorder>
       <zorder>label_4</zorder>
       <zorder>pushButton_2</zorder>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>610</y>
         <width>401</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(170, 0, 0);
background-color: rgb(205, 166, 125);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>2</y>
         <width>701</width>
         <height>731</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Burcas/depositphotos_75177907-stock-illustration-sketchy-beer-and-snacks-seamless.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>label_3</zorder>
      <zorder>frame_3</zorder>
      <zorder>label_2</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagem.qrc"/>
 </resources>
 <connections/>
</ui>
